jira_sync/jira_wrapper.py

```diff
@@ -5,7 +5,7 @@
 """
 import logging
 import re
-from typing import cast, Dict, List, Optional
```

Comment:
I rather use the types provided by typing library as in some cases the standard doesn't work. But I'm not against changing that.

Reply:
The project requires Python 3.11 as a minimum – type hinting generic work since 3.9 and union types (instead of `Optional`) since 3.10.

Reply:
Didn't know about that change in python. Thanks for info